benchmark.go

```diff
@@ -41,6 +42,41 @@ var (
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
 )
 
+// BenchmarkParameters holds values of all benchmark parameters
+type BenchmarkParameters struct {
+	CPUs        int    `json:"cpus"`
+	Concurrency int    `json:"concurrency"`
+	Connections int    `json:"connections"`
+	MaxRequests int    `json:"maxRequests"`
+	MaxDuration string `json:"maxDuration"`
+	MaxRPS      int    `json:"maxRPS"`
+}
+
+// Latencies stores the latencies output for different quantiles
+type Latencies struct {
```

Comment:
related to a previous comment, instead of having a  separate struct for storing Latency values, can we simply use a `map[string]string` and use that in `Summary`